src/core_modules/capture-core/components/ScopeSelector/QuickSelector/OrgUnitSelector.component.js

```diff
@@ -0,0 +1,185 @@
+// @flow
+import React, { Component } from 'react';
+import i18n from '@dhis2/d2-i18n';
+import { withStyles } from '@material-ui/core/styles';
+import Paper from '@material-ui/core/Paper';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import IconButton from '@material-ui/core/IconButton';
+import ClearIcon from '@material-ui/icons/Clear';
+import { Button } from '../../Buttons';
+import { OrgUnitField } from './OrgUnitField.container';
+
+const styles = (theme: Theme) => ({
+    paper: {
+        padding: 8,
+        backgroundColor: theme.palette.grey.lighter,
+    },
+    title: {
+        margin: 0,
+        fontWeight: 425,
+        fontSize: 15,
+        paddingBottom: 5,
+    },
+    form: {
+        width: '100%',
+    },
+    listItem: {
+        backgroundColor: '#ffffff',
+        borderRadius: 5,
+        marginBottom: 5,
+        padding: 5,
+        border: '1px solid lightGrey',
+    },
+    selectedText: {
+        marginTop: 5,
+        marginBottom: 5,
+        padding: 5,
+        borderLeft: '2px solid #71a4f8',
+    },
+    selectedPaper: {
+        backgroundColor: theme.palette.grey.lighter,
+        padding: 8,
+    },
+    selectedButton: {
+        width: 20,
+        height: 20,
+        padding: 0,
+    },
+    selectedButtonIcon: {
+        width: 20,
+        height: 20,
+    },
+    selectedItemContainer: {
+        display: 'flex',
+        alignItems: 'center',
+        minHeight: 28,
+        marginTop: 8,
+        marginBottom: 4,
+        paddingLeft: 5,
+        borderLeft: `2px solid ${theme.palette.primary.light}`,
+    },
+    selectedItemClear: {
+        flexGrow: 1,
+        display: 'flex',
+        justifyContent: 'flex-end',
+    },
+});
+
+type Props = {
+    handleClickOrgUnit: (orgUnitId: ?string, orgUnitObject: ?Object) => void,
+    onReset: () => void,
+    selectedOrgUnitId: string,
+    showWarning: boolean,
+    selectedOrgUnit: Object,
+    classes: Object,
+};
+
+type State = {
+    open: boolean,
+};
+
+class OrgUnitSelectorPlain extends Component<Props, State> {
```

Comment:
Hey @simonadomnisoru , is there any reason why you use class component instead of function component? I think we are moving toward using function component.

Reply:
Hey @huocha,

I wrote in the PR description that the files in this PR are copy-pasted from the `LockerSelector` component. class component was used in these files I didn't change the logic.
Thanks!